runtime/gc_base/GCExtensions.hpp

```diff
@@ -337,6 +344,7 @@ class MM_GCExtensions : public MM_GCExtensionsBase {
 		, minimumFreeSizeForSurvivor(DEFAULT_SURVIVOR_MINIMUM_FREESIZE)
 		, freeSizeThresholdForSurvivor(DEFAULT_SURVIVOR_THRESHOLD)
 		, recycleRemainders(true)
+		, continuationListOption(disable_continuation_list)
```

Comment:
thinking if we have time to default to verify mode first for a few nights, and then disable it